cassandra_nodetool/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,61 @@
+name: Cassandra Nodetool
+files:
+- name: cassandra_nodetool.yaml
+  options:
+  - template: init_config
+    options:
+      - name: nodetool
+        description: |
+          command or path to nodetool (e.g. /usr/bin/nodetool or docker exec container nodetool)
+          can be overwritten on an instance
+          Note: Agent v6.11+ on Windows runs as an unprivileged user (`ddagentuser`). That user needs to be granted
+          access to the nodetool installation directory for the check to work.
+          The nodetool installation also sets some environment variables (e.g. `CASSANDRA_HOME` and `DSCINSTALLDIR`),
+          but sets them as variables only for the user doing the nodetool installation. Those environment variables should
```

Comment:
Do you mean "You should also set those environment variables as system-wide variables."? That's more direct than saying they "should be set."